dj_beneficiary/test_utils/models.py

```diff
@@ -0,0 +1,81 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from dj_beneficiary.models import (
+    BaseAgentAbstract,
+    BaseDistrictAbstract,
+    BaseFacilityAbstract,
+    BaseIndividualBeneficiaryAbstract,
+    BaseOrganizationBeneficiaryAbstract,
+    BaseProvinceAbstract,
+    BaseWardAbstract,
+    ImplementingPartner,
+    FacilityType
+)
+
+
+# Put your test models here
+class Province(BaseProvinceAbstract):
+    pass
+
+
+class District(BaseDistrictAbstract):
+    province = models.ForeignKey(
+        Province,
+        on_delete=models.CASCADE,
+    )
+
+
+class Ward(BaseWardAbstract):
+    district = models.ForeignKey(
+        District,
+        on_delete=models.CASCADE,
+    )
+
+
+class Facility(BaseFacilityAbstract):
+    district = models.ForeignKey(
+        District,
+        default=1,
+        on_delete=models.CASCADE,
+        help_text=_("District in which the facility is located"),
+        max_length=250,
+    )
+    facility_type = models.ForeignKey(
+        FacilityType,
+        default=1,
+        on_delete=models.CASCADE,
+        help_text=_("Facility Type, i.e 'Hospital, Clinic etc"),
+        max_length=250,
+    )
+    implementing_partner = models.ForeignKey(
+        ImplementingPartner,
```

Comment:
this needs to be left on the base abstract for facility as it can be referenced from the abstract class without needing it to berepeated